app/static/src/app/store/run/state.ts

```diff
@@ -22,4 +22,5 @@ export interface RunState {
     userDownloadFileName: string;
     downloading: boolean;
     numberOfReplicates: number;
+    showIndividualTraces: boolean;
```

Comment:
If it does not go into the state, then the check logic gets duplicated twice - that's why I wondered for a bit about doing this via the store. That's complicated a bit further by the nullability of the configuration option. Perhaps this would be best to chat through to make sure we make the least bad choice here

Reply:
I think all that logic could just go into a getter, unless I'm misunderstanding.